fbpic/lpa_utils/laser/laser_profiles.py

```diff
@@ -835,3 +837,175 @@ def E_field( self, x, y, z, t ):
         Ey = self.E0y * profile
 
         return( Ex.real, Ey.real )
+
+class FromLasyFileLaser( LaserProfile ):
+    """Class that emits a laser from a lasy file"""
+
+    def __init__(self, filename, t_start=0.):
+        """
+        Define a laser whose profile is determined by a
+        `lasy <https://lasydoc.readthedocs.io/en/latest/>`_ file.
```

Comment:
```suggestion
        `lasy <https://lasydoc.readthedocs.io/en/latest/>`_ file.
        
        When the laser is initialized by this function, FBPIC forces the beginning of 
        the time axis in the ``lasy`` file to be zero (irrespective of the metadata for 
        ``tmin`` that is actually present in the ``lasy`` file). This convention was chosen
        for convenience, because ``tmin`` in ``lasy`` could otherwise result in large delays
        before emitting the laser, especially when using ``lasy``'s ``propagate`` feature.
```